src/ui/Frame.cpp

```diff
@@ -179,20 +189,26 @@ void Frame::draw(SDL_Rect _size, SDL_Rect _actualSize) {
 		handleRect.y = _size.y + sliderPos;
 		handleRect.w = sliderSize;
 		handleRect.h = handleSize;
+
+		int x = (handleRect.x) * (float)xres / (float)Frame::virtualScreenX;
+		int y = (handleRect.y) * (float)yres / (float)Frame::virtualScreenY;
+		int w = (handleRect.x + handleRect.w) * (float)xres / (float)Frame::virtualScreenX;
+		int h = (handleRect.y + handleRect.h) * (float)yres / (float)Frame::virtualScreenY;
 		if (rectContainsPoint(barRect, omousex, omousey)) {
-			drawWindow(handleRect.x, handleRect.y, handleRect.x + handleRect.w, handleRect.y + handleRect.h);
+			// TODO highlight
+			drawWindow(x, y, w, h);
```

Comment:
so w and h are really like x2 and y2, where x2 = x + w right?

Reply:
yeah I haven't namely a variable x2 or y2 in so long I didn't think to use it!